src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -12,37 +12,59 @@ public class DotnetFirstTimeUseConfigurer
         private IEnvironmentProvider _environmentProvider;
         private INuGetCachePrimer _nugetCachePrimer;
         private INuGetCacheSentinel _nugetCacheSentinel;
+        private IFirstTimeUseNoticeSentinel _firstTimeUseNoticeSentinel;
 
         public DotnetFirstTimeUseConfigurer(
             INuGetCachePrimer nugetCachePrimer,
             INuGetCacheSentinel nugetCacheSentinel,
+            IFirstTimeUseNoticeSentinel firstTimeUseNoticeSentinel,
             IEnvironmentProvider environmentProvider)
         {
             _nugetCachePrimer = nugetCachePrimer;
             _nugetCacheSentinel = nugetCacheSentinel;
+            _firstTimeUseNoticeSentinel = firstTimeUseNoticeSentinel;
             _environmentProvider = environmentProvider;
         }
 
         public void Configure()
         {
-            if(ShouldPrimeNugetCache())
+            if (ShouldPrintFirstTimeUseNotice())
             {
                 PrintFirstTimeUseNotice();
+            }
+
```

Comment:
Any tips on how to capture output in tests? I want to verify whether telemetry was printed or not, but I don't see an obvious API to do that. Any suggestions?